x-pack/plugins/uptime/public/components/functional/filter_bar_loading.tsx

```diff
@@ -15,4 +15,4 @@ const searchBox = {
   }),
 };
 
-export const FilterBarLoading = () => <EuiSearchBar box={searchBox} />;
+export const FilterBarLoading = () => <EuiSearchBar box={searchBox} onChange={()=>{ /* */}}/>;
```

Comment:
I think we should also include a comment explaining that the purpose of this component is to serve as a visual placeholder while the `FilterBar` is awaiting data.

This way, if someone sees this code later and can think of a better solution, they will more easily be able to understand what's happening.
```suggestion
/**
 * This component provides a visual placeholder while the FilterBar is loading.
 * The onChange prop is required, so we provide an empty function to suppress the warning.
 */
export const FilterBarLoading = () => <EuiSearchBar box={searchBox} onChange={()=>{ /* */}}/>;
```